.codeclimate.yml

```diff
@@ -1,5 +1,44 @@
+version: "2"
+
 languages:
   Python: true
+  Javascript: true
+
```

Comment:
is the consensus to put a threshold on _everything_? for some things that the team agrees we don't need, would it make sense to exclude them from being checked? Similar to what frontend does

Reply:
I thought "why not" since there isn't much JS. I did exclude this dir from the tests: "usaspending_api/static_doc_files/js/*"

Reply:
Apologies, I meant to ask about the checks listed below